inc/XamlApplication.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include "XamlApplication.g.h"
+#include <winrt/Windows.UI.Xaml.Hosting.h>
+#include <winrt/Windows.UI.ViewManagement.h>
+#include <winrt/Windows.Foundation.Collections.h>
+#include <Windows.h>
+//#include <DesktopWindow.h>
+namespace winrt::Microsoft::Toolkit::Win32::UI::XamlHost {
+  struct XamlApplication;
+}
+
+namespace winrt::Microsoft::Toolkit::Win32::UI::XamlHost::implementation {
+  enum ExecutionMode {

```

Comment:
`ExecutionMode` can be removed from the design. this library is for xaml islands and UWP can't use those.